### DISCUSSION TITLE
Install Issue

### DISCUSSION THREAD

**Discussion by CR7443** [ReagentX/imessage-exporter#242] [Q&A] [answered]
I am having a hard time getting this installed/Ran in my system. I am new to this world and if there is a dummy version step by step guide of what I would need todo to get this installed that would be amazing.

**Comment by ReagentX** [comment, marked as answer]
Install instructions are located [here](https://github.com/ReagentX/imessage-exporter/blob/develop/imessage-exporter/README.md#installation)!
